### ISSUE TITLE
Apply border radius to .ui.buttons, rather than .ui.button (in container)

### ISSUE THREAD

**Issue by AllanJard** [fomantic/Fomantic-UI#2810]
# Feature Request

I've been using `.ui.buttons { flex-wrap: wrap; }` to allow for button sets which overflow, but the border radius on the buttons causes some issues.

I wonder if it would be possible in `.ui.buttons` to remove the border-radius on `.ui.button` elements and instead apply it to the `.ui.buttons` container.

## Example

Example showing the issue: https://jsfiddle.net/2vu0wxrf/ . It is small, but there is border radius on :first-child and :last-child that is causing the problem.

Updated example with an "important" hack to demonstrate the resolution: https://jsfiddle.net/2vu0wxrf/2/ . 

## Screenshot

![image](https://github.com/fomantic/Fomantic-UI/assets/84905443/609adaf4-3811-40e7-a950-f5ff45c28ab2)

**Comment by lubber-de** [comment]
Good Idea. But using `overflow: hidden` (your trick to make sure the border:0 on each button does not prevent the border on the button group) unfortunately introduces other issues like a tooltip not being fully displayed anymore. Would be the same issue for a dropdown button or other elements inside the button group.

See  https://jsfiddle.net/lubber/wje5cbo7/2/
![image](https://github.com/fomantic/Fomantic-UI/assets/18379884/c40d46e9-99f2-44b9-9d74-71d3ff9b5a2a)

**Comment by AllanJard** [comment]
Bother - good point! Its a shame the border radius from the parent doesn't just propagate down to the children.

That overflow requirement probably makes this a non-starter though. Feel free to close off.

**Comment by lubber-de** [comment]
So, we already have "`wrapping buttons" (which does the same as your custom flex solution).
We also have a helper class "wrapped" which at least adjusts the border radius for top left and bottom right corner

We could think of adopting your addition as a new "always wrapping buttons" adding a new "always" term, which would always show all for corners rounded (beware of some special combinations like "spaced" which should not change the border radius of each eingle button.)

However, this would still need the overflow-hack, so the docs would need to mention that limitation and i wonder if people will even more complain about that when overflowing items inside the menu like tooltips/dropdown menus/etc wont work anymore...
🤔

**Comment by AllanJard** [comment]
Sorry for the delay in getting back to you here! I'd actually missed the `wrapping buttons` option - it's a clever tweak using first- and last-child, but I imagine there might be some who are frustrated about the bottom left and top right not getting a border radius.

The overflow hack works well for simple buttons, but as soon as a dropdown is introduced, the hack is hopeless and that's an important use case for me.

I don't have a good answer unfortunately. Your `wrapping buttons` might be as good as it is possible to get!
